package.json

```diff
@@ -34,9 +39,6 @@
   "dependencies": {
     "@ledgerhq/hw-transport-webusb": "6.27.12"
   },
-  "engines": {
-    "node": ">=20"
```

Comment:
should we not keep engines as well? I think this has a lot of flexibility so we can add a minimum and a maximum. Actually wondering if that would be enough?